charts/rancher-windows-gmsa/102.0.0+up0.6.1/templates/validate-install-crd.yaml

```diff
@@ -0,0 +1,14 @@
+#{{- if gt (len (lookup "rbac.authorization.k8s.io/v1" "ClusterRole" "" "")) 0 -}}
+# {{- $found := dict -}}
+# {{- set $found "windows.k8s.io/v1alpha1/GMSACredentialSpec" false -}}
+# {{- range .Capabilities.APIVersions -}}
+# {{- if hasKey $found (toString .) -}}
+# 	{{- set $found (toString .) true -}}
+# {{- end -}}
+# {{- end -}}
+# {{- range $_, $exists := $found -}}
+# {{- if (eq $exists false) -}}
+# 	{{- required "Required CRDs are missing. Please install the corresponding CRD chart before installing this chart." "" -}}
+# {{- end -}}
+# {{- end -}}
```

Comment:
How come this is all commented out?

Reply:
This is an autogenerated file, I think its because CRD validation isn't set for this chart. So while the file is generated, is commented out since we don't want the logic. You can see the same file in a few other charts used by Rancher too, also commented out